addons/point_of_sale/tests/common_setup_methods.py

```diff
@@ -46,6 +57,7 @@ def setup_product_combo_items(self):
             "available_in_pos": True,
             "list_price": 11,
             "taxes_id": [(6, 0, [tax20in.id])],
+            "pos_categ_ids": [(6, 0, [pos_category_1.id])],
```

Comment:
The category of each item shouldn't be different ?

Reply:
The products I select in the tour have all different categories.